src/Xamarin.Android.Build.Tasks/Tasks/ConvertCustomView.cs

```diff
@@ -75,7 +78,7 @@ public override bool Execute ()
 					}
 				}
 			}
-
+			Processed = processed.Select (x => new TaskItem (x)).ToArray ();
```

Comment:
Can we use a `List<ITaskItem>` here to avoid the `System.Linq` and `Select()`?

Reply:
I'll change it too a foreach loop.